js/app/modules/contentGroup/contentGroup.js

```diff
@@ -217,6 +227,30 @@ const ContentGroup = new Module.Class({
         if (!this._selection.in_error_state)
             return;
         this._stack.visible_child_name = ERROR_PAGE_NAME;
+        this._log_button_stack.visible_child_name = 'button';
+    },
+
+    _on_log_button_click: function () {
+        let timestamp = new Date(Date.now());
+        let stamp = 'eos-knowledge-lib log ' + timestamp.toISOString() + '.txt';
+        let log_name = GLib.build_filenamev([GLib.get_home_dir(), stamp]);
```

Comment:
I like the idea of putting it on /tmp or a tmpfs mounted path. This way we don't need to grant write permissions for the whole users home, or worried about trashing it.

Once opened, the user can copy/paste or save it anywhere he wants.